modules/010-deckhouse-flow-schema/templates/flowschema.yaml

```diff
@@ -0,0 +1,191 @@
+# Schema generated by "tools/flow_schema.go" DO NOT EDIT.
+# To generate run 'make generate'
+---
+apiVersion: flowcontrol.apiserver.k8s.io/v1beta2
+kind: FlowSchema
+metadata:
+  name: deckhouse-flow-schemas
+  {{- include "helm_lib_module_labels" (list . (dict "app" .Chart.Name)) | nindent 2 }}
+spec:
+  distinguisherMethod:
+    type: ByUser
+  matchingPrecedence: 1000
+  priorityLevelConfiguration:
+    name: deckhouse-prioritylevel
+  rules:
+  - resourceRules:
```

Comment:
NIT: You can save two code blocks by adding API groups to a single array.

```yaml
    - apiGroups: ["",  "deckhouse.io", "cilium.io"]
      clusterScope: true
      namespaces: ["*"]
      resources: ["*"]
      verbs: ["list"]
```

Reply:
good point, thanks